src/services/UsersService.ts

```diff
@@ -0,0 +1,40 @@
+import { inject, injectable } from "inversify";
```

Comment:
different quotes in import. You should follow only one qoute style through all project